_includes/_help/search/_posts/second-order.md

```diff
@@ -30,10 +30,10 @@ will return a ranked list of papers spanning a variety of topics useful to resea
 
 will return a ranked list of papers featuring reviews of weak gravitational lensing and its cosmological implications.
 
-<figure>
+{% if include.ads %}<figure>
```

Comment:
I'd actually keep this figure on the SciX side - there's nothing ADS-specific about it.